vosekast_control/I2cInputAndRelayControl.py

```diff
@@ -0,0 +1,70 @@
+import smbus
+from typing import Dict, List
+
+
+class RelayControl:
+    def __init__(self):
+        self.address_relays = 0x38
+        self.bus = smbus.SMBus(1)
+        self.state_dict = {}
```

Comment:
you use the `state_dict` only in a getter function without caching. So to store that variable in the class context makes no sense.